web/templates/stream_settings/announce_stream_checkbox.hbs

```diff
@@ -1,5 +1,5 @@
-<label class="checkbox">
-    <input type="checkbox" name="announce" value="announce" checked />
+<label class="checkbox" for="{{#if prefix}}{{prefix}}{{/if}}{{setting_name}}">
+    <input type="checkbox" name="announce" value="announce" checked id="{{#if prefix}}{{prefix}}{{/if}}{{setting_name}}"/>
```

Comment:
Can we just directly define ID here instead of passing the prefix and setting name as variables?

Also, there is a bug when announcements stream setting is updated.